### DISCUSSION TITLE
Token healing?

### DISCUSSION THREAD

**Discussion by yonitjio** [guidance-ai/guidance#752] [Q&A] [answered]
I'm getting a '&lt;s&gt;' in my text generation and I thought this can be fixed with token healing, but I can't find token_healing anywhere in the source code, is it removed or it's a fixed feature now?

What should I do?

**Comment by riedgar-ms** [comment]
Can you show a full repro case, please?

**Reply by yonitjio** [reply, marked as answer]
Hi, sorry for the very late reply.

I experienced this when I was experimenting with guidance library to create a chat bot.
I can't give you a full repro case because it's random occurrences. 

And since it's a minor issue with my experiment, I just stripped it out and never look at it again.
```
sql = re.sub(r'(<s>|</s>)', '', sql).strip()
```

If you want to take a look at my source it's [here](https://github.com/yonitjio/exploring-odoo/blob/main/ai_bot/models/ai/odoo_ai.py#L117)

But I doubt you want to install the entire software to test it, so I'm gonna close this for now.

Thanks.
